src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -713,7 +716,7 @@ private void ProcessNode(ref BioProcesses processor, ref CompoundStorage storage
             // For now lets assume compounds we produce are also useful
             bag.SetUseful(entry.Key);
 
-            var outputAdded = entry.Value * process.Rate * environmentModifier;
+            var outputAdded = entry.Value * process.Rate * environmentModifier * process.SpeedMultiplier;
```

Comment:
This is not fully correct as this just multiplies the outputs, and not the inputs.

Reply:
This is still the case, you only added the multiplier check to the input validation, the input consuming loop (which is on line 797) is not yet updated.